auto_merger/merger.py

```diff
@@ -54,8 +54,8 @@ def __init__(self, config: Config):
         self.approvals = self.config.github["approvals"]
         self.namespace = self.config.github["namespace"]
         self.pr_lifetime = self.config.github["pr_lifetime"]
-        self.pr_to_merge = {}
-        self.approval_body = []
+        self.pr_to_merge: Dict = {}
+        self.approval_body: List = []
```

Comment:
You can use `list` and `dict` and ideally specify also the expected content. See this useful cheat sheet: https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html